messages/common.options

```diff
@@ -14,3 +14,7 @@
 
 PubResponse.pub max_size:500
 RootFingerprintResponse.fingerprint fixed_length:true max_size:4
+XPub.depth fixed_length:true max_size:1
+XPub.parent_fingerprint fixed_length:true max_size:4
+XPub.chain_code fixed_length:true max_size:32
+XPub.public_key fixed_length:true max_size:33
```

Comment:
Why 33 (please write a comment)? and missing newline.

Reply:
Done